### PR TITLE
Welcome and sign-in flow updates

### PR DESCRIPTION
**Pull request by jedireza**
In this PR we:
* Update the welcome and sign-in flows with updated copy and visuals
* Update the premium screen copy under settings
* Update the zero query stub queries
* Calculate the percent off between annual and monthly plans instead of using static text
* Clean up some old unused functions

### How was this tested?
- [x] I tested this manually

### Manual test cases
Simulator; iphone+ipad

#### iPhone

| Light | Dark |
|---|---|
| image | image |

#### iPad

| Light | Dark |
|---|---|
| image | image |

## Issues addressed
🔒 https://www.figma.com/file/6BWVIuiRpXepCbdZvlKK16/First-run---w-NeevaAI-messaging?node-id=0%3A1&t=lhKCiaXDLXbwZOND-0
🔒 https://app.clickup.com/t/36034046/BEE-2483

### CODE REVIEW

**Comment by seth737** [comment]
Task linked: [BEE-2483 Changes to first run (ios browser)](https://app.clickup.com/t/36034046/BEE-2483)

**Review by yozuysal — APPROVED** [review]
thank you!

File: Client/Frontend/Settings/Neeva/NeevaPremiumView.swift [hunk 1]
```diff
@@ -28,15 +28,17 @@ struct NeevaPremiumView: View {
     }
 
     var premiumBullets = [
-        ("Browser + ad blocker + tracking prevention", ""),
-        ("Unlimited ad-free, private search", ""),
+        ("Private browser with ad blocker & tracking prevention", ""),
```
Comment (chung-lai) [inline Client/Frontend/Settings/Neeva/NeevaPremiumView.swift:31]: I believe these strings need to be replaced in localizable.strings file for translation.

**Review by chung-lai — COMMENTED** [review]
Besides the localization comment, everything else LGTM!

File: Client/Frontend/SignInOrUpFlow/SignInOrUpFlowPlansView.swift [hunk 2]
```diff
@@ -264,29 +268,24 @@ struct SignInOrUpFlowPlansView: View {
...
-                        Text("Already have an account? Sign in")
-                            .withFont(.labelLarge)
-                            .foregroundColor(.secondary)
-                            .frame(maxWidth: .infinity)
+                        HStack {
+                            Group {
+                                Text("Already have an account?")
+                                    .foregroundColor(.secondary)
+                                Text("Sign in")
+                                    .foregroundColor(.primary)
```
[the remark this answers is not in the document]
Comment (jedireza) [reply Client/Frontend/SignInOrUpFlow/SignInOrUpFlowPlansView.swift:276]: We chatted in Slack about these and will ignore for now.

**Pull request merged**